daemon-tests/src/lib.rs

```diff
@@ -418,7 +418,7 @@ pub fn dummy_new_order(position: Position) -> maker_cfd::NewOrder {
         // 8.76% annualized = rate of 0.0876 annualized = rate of 0.00024 daily
         funding_rate: FundingRate::new(dec!(0.00024)).unwrap(),
         opening_fee: OpeningFee::new(Amount::from_sat(2)),
-        position,
+        position_maker: position,
```

Comment:
🔧 Since you're making this little change, should we do this too?

```suggestion
        position_maker,
```